### ISSUE TITLE
Managed Calendars for Series.

### ISSUE THREAD

**Issue by svevang** [PRX/cms.prx.org#516]
Some feeder Podcasts lack a corresponding Series in CMS. New episodes for these
podcasts are released via direct feeder access.

In general we need a way to manage calendar data for all our shows that either
lack a publish workflow or don't bother to update their calendar. So that every podcast in feeder find it's own calendar.

Create a 'managed' calendar for these podcasts. The entries in these managed
calendars (minimally implementing the `published_released_at` semantics) would be hidden from publish users but be present in the calendar endpoints.

**Comment by brandonhundt** [comment]
Do we need a production calendar for TAL? --- We don't do sales for them.

**Comment by svevang** [comment]
Here are the nine podcasts on feeder that lack a prx_uri:

```
[[12, "Serial Test"],
[13, "Serial"],
[45, "S-Town"],
[21, "Demo"],
[23, "99% Invisible"],
[138, "Dog Talk"],
[33, "The Bugle"],
[22, "Love + Radio"],
[188, "This American Life"]]
```

**Comment by cavis** [comment]
Makes sense.  Serial/S-Town/TAL are manually set to feeder.  That 99pi is the old feed, though it's still getting traffic for some reason.  And Bugle/L+R are likely still on wordpress right now (and coming in via [crier.prx.org](https://github.com/PRX/crier.prx.org)).
